sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -143,7 +143,10 @@ def callback(raw_response, _, headers):  # pylint: disable=unused-argument
         if model_id is None:
             model_id = str(uuid.uuid4())
 
-        return self._client.begin_build_document_model(  # type: ignore
+        _client_op_path = self._client
+        if self._api_version == DocumentAnalysisApiVersion.V2023_02_28_PREVIEW:
+            _client_op_path = _client_op_path.document_models
+        return _client_op_path.begin_build_model(  # type: ignore
```

Comment:
This previously was begin_build_document_model for 2022-08-31

Reply:
Done! Thanks for pointing that out, we'll do a special check for the august version and otherwise continue with the new preview operations.